### ISSUE TITLE
Hey Chichou, what is this meant to be?

### ISSUE THREAD

**Issue by Forterc** [ChiChou/vscode-frida#2]
I'd like to contribute, but im unsure what exactly are you trying to do. Are you making a frida Vscode extension?

**Comment by ChiChou** [comment]
Yep. I am thinking about the best way to intergrate javascript editor to frida REPL

**Comment by ChiChou** [comment]
Something like Jupyter Notebook, maybe

**Comment by Forterc** [comment]
Sounds cool. I can try and make a base template readme, or if you'd like me to help with bigger stuff just detail the issues and i'll try to take what I can from there.

**Comment by ChiChou** [comment]
Besides I'm about to intergrate my project [Grapefruit](https://github.com/chichou/grapefruit) via WebView

**Comment by Forterc** [comment]
Why would I use grapefruit over frida for iOS?

**Comment by ChiChou** [comment]
> Why would I use grapefruit over frida for iOS?

It’s just an option. I’ve already integrated Objection

**Comment by Forterc** [comment]
Nice stuff. I saw your twitter demo, which seems pretty functional: https://twitter.com/CodeColorist/status/1223994354479419392

Are there any major issues you'd like contribution with?

**Comment by ChiChou** [comment]
* Android support (I don't use it but I'd like it to be compatible)
* Debugging support
* `frida-compile` support for TypeScript projects
